pkg/rancher-desktop/main/extensions/extensions.ts

```diff
@@ -324,11 +334,24 @@ export class ExtensionImpl implements Extension {
       {
         name:      this.containerName,
         namespace: this.extensionNamespace,
-        env:       { DESKTOP_PLUGIN_IMAGE: this.id },
+        env:       { DESKTOP_PLUGIN_IMAGE: this.image },
       },
     );
   }
 
+  get isInstalled(): Promise<boolean> {
```

Comment:
I don't see the advantage of making this a getter. The name assumes it's a method, and making it async makes it more method-like. If getters' return values were cached I'd say otherwise, but they aren't, right?